client/src/containers/Category.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Category = () => {
+    return (
+        <div>
+            <h1> This is a category page which will list down all the categories with around 5 blogs each</h1>
+        </div>
+    )
+}
+
+export default Category;
```

Comment:
Add new line here

Reply:
I am going to work on category in next PR